products/src/components/ProductList.js

```diff
@@ -53,8 +53,9 @@ class ProductList extends React.Component {
                 <div className='container-fluid'>
                     <h3 className='p-2'>Products</h3>
                     <Link to="/add"><Button varient="outline-secondary" size="sm">Add Product</Button></Link>
-                    <Table hover>
-                        <thead>
+                    <div style={{'height':'72vh','overflow-y':'scroll'}}>
```

Comment:
Please move the inline css to a file and import and use it.

Reply:
Ok. I will address.